frontends/p4/tableApply.cpp

```diff
@@ -16,12 +16,13 @@ limitations under the License.
 
 #include "tableApply.h"
 
+#include "frontends/common/resolveReferences/referenceMap.h"
```

Comment:
Why does this need to be here? It's already included via tableApply.h

Reply:
I had a work-in-progress PR that runs IWYU across the entire codebase to clean up includes. 
https://github.com/p4lang/p4c/pull/3767/

I am hesitant to push it further because IWYU still seems to immature as a tool. Wish clang-tidy would make its include-fixer an independent tool...